docs/source/jupyterhub.rst

```diff
@@ -280,32 +284,26 @@ Sharing notebooks
 
 
 iRODS
-^^^^^
-
-Open a new terminal on your Jupyter and follow this steps:
-1. Install icommands:
+-----------------
 
-.. code-block:: bash
+iRODS support is provided using a wrapper script and a singularity image already copied to your home profile. 
+Before start using iRODS, you'll need to copy your environment file from the farm to your jupyter. Open a Terminal and please follow this steps:
 
-    wget https://files.renci.org/pub/irods/releases/4.1.10/ubuntu14/irods-icommands-4.1.10-ubuntu14-x86_64.deb
-    apt-get install ./irods-icommands-4.1.10-ubuntu14-x86_64.deb
+1. Use ``mount-farm`` and input your credentials when promted.
  
-2. Create the ``.irods`` folder on your home directory:
+2. Copy ``irods_environment.json`` from your home directory on the farm to your Jupyter instance:
 
 .. code-block:: bash
 
-    mkdir -p ~/.irods
- 
-3. Copy your irods_environment.json from your home directory on the farm to your Jupyter instance:
-
-.. code-block:: bash
+    cp /nfs/users/nfs_a/ak27/.irods/* ~/.irods/
```

Comment:
it says `ak27`, is it correct?

Reply:
Yes, all examples that involve the farm or nfs use `ak27` as de sample user.

Reply:
Ok then!